tensorboard/plugins/histogram/summary.py

```diff
@@ -36,12 +36,12 @@
 
 
 # Export V2 versions.
+histogram_v2 = summary_v2.histogram_v2
+
+# Export the default versions.
 histogram = summary_v2.histogram
 histogram_pb = summary_v2.histogram_pb
 
-# Export V3 versions.
-histogram_v3 = summary_v2.histogram_v3
```

Comment:
Let's keep this one line for now, so that anyone who was testing this out internally won't be broken by this change?

Reply:
Done.